### ISSUE TITLE
`NO_COLOR=1` does not works for "Did you mean ..."

### ISSUE THREAD

**Issue by lambdalisue** [denoland/deno#12118]
It seems `NO_COLOR=1` does not affect coloring on "Did you mean ..." part.

![CleanShot 2021-09-17 at 15 52 20](https://user-images.githubusercontent.com/546312/133737817-fa16076d-9725-4463-9a65-e5aab8cbf98d.png)

My Deno version is 

```
deno 1.14.0 (release, x86_64-apple-darwin)
v8 9.4.146.15
typescript 4.4.2
```

Thanks for the project 👍

**Comment by leizongmin** [comment]
I think this a bug of the `clap` crate. I have just submitted an issue on https://github.com/clap-rs/clap/issues/2776

**Comment by leizongmin** [comment]
The `clap` crate has already  fixed this problem since `3.0.0-beta.1`, we should wait for the `clap@3.0.0` release.

**Comment by crowlKats** [comment]
Related to #8475 then
